Python/PRP/GUI/pyGUIListBoxMod.cpp

```diff
@@ -0,0 +1,93 @@
+/* This file is part of HSPlasma.
+*
+* HSPlasma is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+*
+* HSPlasma is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with HSPlasma.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include <PRP/GUI/pfGUIListBoxMod.h>
+#include "pyGUIListBoxMod.h"
+#include "PRP/KeyedObject/pyKey.h"
+#include "PRP/GUI/pyGUIControlMod.h"
+
+extern "C" {
+
+PY_PLASMA_VALUE_NEW(GUIListElement, pfGUIListElement)
+
+PY_PROPERTY(bool, GUIListElement, selected, isSelected, setSelected)
+
+static PyGetSetDef pyGUIListElement_GetSet[] = {
+    pyGUIListElement_selected_getset,
+    PY_GETSET_TERMINATOR
+};
+
+PY_PLASMA_TYPE(GUIListElement, pfGUIListElement, "pfGUIListElement wrapper");
+
+PY_PLASMA_TYPE_INIT(GUIListElement) {
+    pyGUIListElement_Type.tp_new = pyGUIListElement_new;
+    pyGUIListElement_Type.tp_getset = pyGUIListElement_GetSet;
+    if (PyType_Ready(&pyGUIListElement_Type) < 0)
+        return NULL;
+
+    /* Konstants */
+    PY_TYPE_ADD_CONST(GUIListElement, "kText", pfGUIListElement::kText);
+    PY_TYPE_ADD_CONST(GUIListElement, "kPicture", pfGUIListElement::kPicture);
+    PY_TYPE_ADD_CONST(GUIListElement, "kTreeRoot", pfGUIListElement::kTreeRoot);
+
+    Py_INCREF(&pyGUIListElement_Type);
+    return (PyObject*)&pyGUIListElement_Type;
+}
+
+PY_PLASMA_VALUE_IFC_METHODS(GUIListElement, pfGUIListElement)
+
+}
+
+
+extern "C" {
+
+PY_PLASMA_NEW(GUIListBoxMod, pfGUIListBoxMod)
+
+PY_PROPERTY(plKey, GUIListBoxMod, scrollControl, getScrollCtrl, setScrollCtrl)
```

Comment:
I wasn't sure if we had a preference for that.  I was trying to expand the names for Python if doing so wasn't unreasonably long, but Ctrl is standard enough I think it would work to shorten it here too.